cypress/support/commands.ts

```diff
@@ -0,0 +1,59 @@
+/// <reference types="cypress" />
+// ***********************************************
+// This example commands.ts shows you how to
+// create various custom commands and overwrite
+// existing commands.
+//
+// For more comprehensive examples of custom
+// commands please read more here:
+// https://on.cypress.io/custom-commands
+// ***********************************************
+//
+//
+// -- This is a parent command --
+// Cypress.Commands.add('login', (email, password) => { ... })
+//
+//
+// -- This is a child command --
+// Cypress.Commands.add('drag', { prevSubject: 'element'}, (subject, options) => { ... })
+//
+//
+// -- This is a dual command --
+// Cypress.Commands.add('dismiss', { prevSubject: 'optional'}, (subject, options) => { ... })
+//
+//
+// -- This will overwrite an existing command --
+// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+//
+// declare global {
+//   namespace Cypress {
+//     interface Chainable {
+//       login(email: string, password: string): Chainable<void>
+//       drag(subject: string, options?: Partial<TypeOptions>): Chainable<Element>
+//       dismiss(subject: string, options?: Partial<TypeOptions>): Chainable<Element>
+//       visit(originalFn: CommandOriginalFn, url: string, options: Partial<VisitOptions>): Chainable<Element>
+//     }
+//   }
+// }
+import { auth, attachEmulators } from '../../src/data/firebase';
+
+attachEmulators(); // Make sure the emulators have been attached when Cypress accesses them.
+
+Cypress.Commands.add('deleteUser', () => {
+  auth.currentUser.delete();
+});
+
+Cypress.Commands.add('clearDb', () => {
+  fetch("http://127.0.0.1:8080/emulator/v1/projects/loggr-a3f89/databases/(default)/documents", {
```

Comment:
Get `loggr-a3f89` out of config instead. Hard-coding this could lead to hard-to-debug test failures.